docs/best-practices.md

```diff
@@ -274,6 +276,38 @@ Checking system health.
 ....
 ```
 
+## Attributes
+
+Reference: [`Document Attributes`](https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/)
+Reference: [`Inline Passthroughs`](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/)
+
+Attributes are a way of creating a variable with a content which then can be used throughout the document. Using attributes increases
+readability a lot and makes it easy to change content one time for all locations used. Attributes are written on top of the page
+and have folloing form:
+
+Definition:
+```
+:name-of-an-attribute: value
+```
+
+Usage:
+```
+This is some text {name-of-an-attribute} which continues here.
+```
+
+Attributes can also be used in scripts when you want to handover common used values. In case, you must define that attributes are
```

Comment:
```suggestion
Attributes can also be used in scripts when you want to handover common used values. In case, you must define that attributes and macros
```